source/postcard-c/example.c

```diff
@@ -0,0 +1,121 @@
+#include "postcard.h"
```

Comment:
There should probably be an SPDX header here. I do plan to use Postcard-C in projects where this is required.

Reply:
Yeah that makes sense. I can add a SPDX header for MIT + Apache dual license to match the rest of the repo